Plots/Panels/NCL_panel_20.py

```diff
@@ -0,0 +1,223 @@
+"""
+NCL_panel_20.py
+===============
+This script illustrates the following concepts:
+   - Drawing four different-sized plots on the same page using gridspec
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: http://www.ncl.ucar.edu/Applications/Scripts/panel_20.ncl
```

Comment:
```suggestion
   - Original NCL script: http://www.ncl.ucar.edu/Applications/Scripts/panel_20.ncl
```
Indent is one space too far right for this line and line 9